webpack/ForemanTasks/Components/TaskDetails/Components/Task.js

```diff
@@ -25,6 +25,7 @@ const Task = props => {
     }
     unlockTaskRequest(id, action);
   };
+
```

Comment:
can you squash the commit which is responsible for it?

Reply:
can you squash all of the commits actually?

Reply:
Looks like I didn't rebase, only the last 2 commits are relevant, should I squash them?